### DISCUSSION TITLE
Add one-click shortcut by section headers to easily create links to that anchor

### DISCUSSION THREAD

**Discussion by shaunagm** [knowledgefutures/pubpub#2121] [Ideas]
When I want to link someone to a specific section of a pubpub page, I need to go to the page, click inspect, grab the generated hash for the header, and then manually add that hash to the url. Conversely, when I link to a subsection of a Github readme or on many other sites, I can just clink the little link symbol by a section header to create a deep link to that anchor. (For example, [here](https://github.com/pubpub/pubpub#to-run-dev-mode).)

This is not merely a matter of convenience for me. For other people in my community, they simply can't make deep links at all, since the steps starting with "inspect the page's source" are too opaque to them. So this would be a huge value add for us.

**Comment by gabestein** [comment]
Hi Shauna — to clarify, you're looking for this on pages, not pubs, correct?

**Reply by shaunagm** [reply]
Yes, exactly. I haven't tried to do this with pubs yet, but we definitely need it on pages.

**Reply by gabestein** [reply]
Gotcha, thanks. We have it on Pubs, and they use the same editor as Pages behind the scenes, so I suspect (but can't say for certain) that it wouldn't be too difficult. We'll chat at next week's dev sync and follow up when we know more!
